src/App.tsx

```diff
@@ -1,15 +1,15 @@
-import React from "react";
+import React, { FC } from "react";
 import { Main } from "./pages/Main";
 import { DeskproAppProvider } from "@deskpro/app-sdk";
 import "iframe-resizer/js/iframeResizer.contentWindow.js";
 
 import "@deskpro/deskpro-ui/dist/deskpro-ui.css";
 import "@deskpro/deskpro-ui/dist/deskpro-custom-icons.css";
 
-const App = () => {
+const App: FC = () => {
```

Comment:
I see that you use `FC` quite a bit, but I would recommend that you don't. It offers no real advantages and has a handful of disadvantages: 

https://github.com/facebook/create-react-app/pull/8177#issue-353062710